lib/wallaby/experimental/selenium/webdriver_client.ex

```diff
@@ -316,4 +364,15 @@ defmodule Wallaby.Experimental.Selenium.WebdriverClient do
       driver: Wallaby.Experimental.Selenium,
     }
   end
+
+  @doc """
+  Retrieves the text from an alert, prompt or confirm.
+  """
+  @spec alert_text(Session.t) :: {:ok, String.t}
+  defp alert_text(session) do
```

Comment:
We either need to make this public or remove the doc. Otherwise we get a compiler warning which is currently whats breaking your build. I'm ok with making it public if we want to do that.